### PR TITLE
Add ffmpeg-shared version 4.1.3

### PR DESCRIPTION
**Pull request by StelFux**
Hello,
I need the ddl libraries with ffmpeg, which are not provided in the stastic version, for Audacity.

### CODE REVIEW

File: bucket/ffmpeg-shared.json [hunk 1]
```diff
@@ -0,0 +1,35 @@
+{
+    "version": "4.1.3",
+    "homepage": "https://ffmpeg.zeranoe.com/builds/",
+    "license": "GPL-3.0",
```
Comment (Ash258) [inline bucket/ffmpeg-shared.json]: Wrong identifier

File: bucket/ffmpeg-shared.json [hunk 2]
```diff
@@ -0,0 +1,35 @@
+{
+    "version": "4.1.3",
+    "homepage": "https://ffmpeg.zeranoe.com/builds/",
```
Comment (Ash258) [inline bucket/ffmpeg-shared.json]: No description

**Comment by r15ch13** [comment]
https://github.com/lukesampson/scoop/pull/3404

File: bucket/ffmpeg-shared.json (hunk 1 above)
Comment (StelFux) [reply bucket/ffmpeg-shared.json]:
```suggestion
    "license": {
        "identifier": "GPL-3.0-only"
    },
```
Is it better this way?

Comment (r15ch13) [reply bucket/ffmpeg-shared.json]: Already fixed with https://github.com/scoopinstaller/scoop-main/commit/ab92aa791c959d26be824986844d571c2fc45286 :smile:

Comment (Ash258) [reply bucket/ffmpeg-shared.json]: No. Do not use object unless you put URL.

Just use `"license": "GPL-3.0-only"`

**Pull request merged**